### PR TITLE
T: Simplify DemorgansLawIntentionTest

### PR DESCRIPTION
**Pull request by dima74**

### CODE REVIEW

File: src/test/kotlin/org/rust/ide/intentions/DemorgansLawIntentionTest.kt [hunk 1]
```diff
@@ -37,84 +37,44 @@ class DemorgansLawIntentionTest : RsIntentionTestBase(DemorgansLawIntention::cla
...
         fn main() {
             if !(20 < 50 /*caret*/|| 40 == 20) {}
         }
-    """, """
-        fn main() {
-            if 20 >= 50 && 40 != 20 {}
-        }
     """)
 
     fun `test constant`() = doAvailableTest("""
```
Comment (ortem) [inline src/test/kotlin/org/rust/ide/intentions/DemorgansLawIntentionTest.kt:80]: Just wondering, what about converting other tests to symmetric ones?

**Review by ortem — APPROVED** [review]
LGTM except for one minor question

File: src/test/kotlin/org/rust/ide/intentions/DemorgansLawIntentionTest.kt (hunk 1 above)
Comment (dima74) [reply src/test/kotlin/org/rust/ide/intentions/DemorgansLawIntentionTest.kt]: Converted

**Comment by dima74** [comment]
bors r=ortem

**Comment by dima74** [comment]
bors retry

**Pull request merged**